src/cmd/build.js

```diff
@@ -37,14 +39,55 @@ export type PackageCreatorParams = {
   artifactsDir: string,
 };
 
+export type LocalizedNameParams = {
+  messageFile: string,
+  manifestData: ExtensionManifest,
+}
+
 export type PackageCreatorFn =
     (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
 
+export async function getDefaultLocalizedName(
+  {messageFile, manifestData}: LocalizedNameParams
+): Promise<string> {
+
+  let messageData: string|Buffer;
+  let extensionName: string = manifestData.name;
+
+  try {
+    messageData = await fs.readFile(messageFile);
+  }
+  catch (err) {
+    log.info(err);
+    throw new WebExtError(`${err}`);
+  }
+
+  try {
+    messageData = JSON.parse(messageData);
+  }
+  catch (err) {
+    throw new WebExtError(`The JSON file is malformed: ${err}`);
```

Comment:
This can be a `UsageError` since it's fixable. This message needs to include the filename so that the developer can track down and fix the issue. The same comment applies to the error up above so I think you can just group these together like:
````
try {
  messageData = JSON.parse(await fs.readFile(messageFile));
} catch (error) {
  throw new UsageError(`Error reading or parsing message file ${messageFile}: ${error}`);
}
````
I also changed `err` to `error` because I think it's easier to read. Usually abbreviations are fine but only if they significantly cut down on the number of characters needed otherwise.